scripts/get_cmake_vars/CMakeLists.txt

```diff
@@ -109,7 +109,11 @@ macro(_vcpkg_adjust_flags flag_var)
             endif()
             string(APPEND ${flag_var} " -isysroot ${CMAKE_OSX_SYSROOT}")
             if (CMAKE_OSX_DEPLOYMENT_TARGET)
-                string(APPEND ${flag_var} " -mmacosx-version-min=${CMAKE_OSX_DEPLOYMENT_TARGET}") 
+                if(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
+                    string(APPEND ${flag_var} " -mmacosx-version-min=${CMAKE_OSX_DEPLOYMENT_TARGET}") 
+                elseif(CMAKE_SYSTEM_NAME STREQUAL "iOS")
+                    string(APPEND ${flag_var} " -mios-version-min=${CMAKE_OSX_DEPLOYMENT_TARGET}") 
+                endif()

```

Comment:
@m-kuhn isn't there a better way to add this ? I remember seeing something in the cmake platform cmake files which looks extremely similar

Reply:
https://github.com/Kitware/CMake/blob/master/Modules/Platform/Apple-Clang.cmake

Reply:
i.e. replace it with ?
`CMAKE_C_OSX_DEPLOYMENT_TARGET_FLAG` instead of an if ?